mopro-core/Cargo.toml

```diff
@@ -44,8 +44,14 @@ ark-zkey = { path = "../ark-zkey" }
 thiserror = "=1.0.39"
 color-eyre = "=0.6.2"
 criterion = "=0.3.6"
+jemalloc-ctl = "0.5.4"
```

Comment:
Can we hide these behind a feature flag? say `gpu-benchmarks`. This way we can be more aggressive with dependencies etc without bloating up library size for other consumers

Reply:
Done. I put them behind the `gpu-benchmarks` flag and revised related code in both `mopro-core` and `mopro-ffi`